faunadb-java-dsl/src/main/java/com/faunadb/client/types/Value.java

```diff
@@ -726,14 +728,11 @@ public int hashCode() {
 
     @Override
     public String toString() {
-      String str = FluentIterable.from(asList(value)).transform(new Function<Byte, String>() {
-        @Override
-        public String apply(Byte input) {
-          return format("0x%02x", input);
-        }
-      }).join(on(", "));
-
-      return format("BytesV(%s)", str);
+     String str = asList(value).stream()
```

Comment:
I believe`Arrays.stream(value)` would be better to initialize a stream

Reply:
There isn't a variant of `Arrays.stream` that takes a `byte[]`.